CMakeLists.txt

```diff
@@ -403,10 +403,15 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "Linux")
 	list(APPEND LIBS rt)
 endif()
 
+set(USE_GL 1)
+
 if(ENABLE_HEADLESS)
-	message(STATUS "Enabling Headless! Disabling GUI, force enabling EGL!")
+	message(STATUS "Enabling Headless, disabling GUI!")
 	set(USE_X11 0)
-	set(USE_EGL 1)
+  if(NOT USE_EGL)
+    set(USE_GL 0)
```

Comment:
Tabs here please 😢 
Old CMake code is still using them and we need to keep it consistent.